woodwork/utils.py

```diff
@@ -115,6 +124,10 @@ def read_file(filepath=None,
     if content_type not in type_to_read_func_map:
         raise RuntimeError('Reading from content type {} is not currently supported'.format(content_type))
 
+    if content_type == 'application/parquet':
```

Comment:
It might be worthwhile to note that `guess_type` won't infer the type of a parquet file automatically. I think we could add it ourselves though using [`mimetypes.add_type`](https://docs.python.org/3/library/mimetypes.html#mimetypes.add_type), but users can also just manually specify the type as well instead.

Reply:
I'm fine with either option here. @gsheni Do you have a preference?